bundles/automation/org.eclipse.smarthome.automation.module.core/META-INF/MANIFEST.MF

```diff
@@ -5,6 +5,7 @@ Bundle-SymbolicName: org.eclipse.smarthome.automation.module.core
 Bundle-Version: 0.9.0.qualifier
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Import-Package: com.google.common.collect,
+ org.apache.commons.lang,
```

Comment:
Sure, it fully makes sense. 

As the automation parts happen to be used in other products as well, I just want to be sure this does not add any additional burden to the maintainers.